collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/SwitchProjectTest.kt

```diff
@@ -1,59 +1,45 @@
 package org.odk.collect.android.feature.projects
 
-import org.junit.After
-import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.RuleChain
 import org.odk.collect.android.R
-import org.odk.collect.android.application.Collect
-import org.odk.collect.android.injection.DaggerUtils
-import org.odk.collect.android.projects.Project
 import org.odk.collect.android.support.CollectTestRule
 import org.odk.collect.android.support.TestRuleChain
 
 class SwitchProjectTest {
 
-    private val projects = listOf(
-        Project("Turtle nesting", "T", "#00FF00", "1"),
-        Project("Polio - Banadir", "P", "#FF0000", "2")
-    )
-
     val rule = CollectTestRule()
 
     @get:Rule
     var chain: RuleChain = TestRuleChain
         .chain()
         .around(rule)
 
-    @Before
-    fun setup() {
-        val projectsRepository = DaggerUtils.getComponent(Collect.getInstance()).projectsRepository()
```

Comment:
I could have sworn I wrote a comment about this in a review, but apparently I didn't. My fault! 

Although it can speed up tests I strongly feel it's best to avoid interacting with our "data" layer in app level tests. There are a couple of reasons for this:

* It bypasses the "real" way state/data is set up when a user is interacting with the app. This means the thing you're testing could potentially only work because the code in your test setup is correct, whereas the flow may be broken in real life.
* It means the tests have to change if there are changes to the domain objects, repositories etc. This can look a little confusing in history, but more importantly means that someone changing the test would then have to have knowledge of how to update the setup code, so it creates a realistic simulation of the user flow.